src/Array.ts

```diff
@@ -42,6 +44,27 @@ export const getMonoid = <A = never>(): Monoid<Array<A>> => {
   }
 }
 
+export const getSetoid: <A>(S: Setoid<A>) => Setoid<A[]> = getArraySetoid
+
+export const getOrd = <T>(O: Ord<T>): Ord<Array<T>> => ({
```

Comment:
Just a nitpick: maybe we can rename the type parameter `T` to `A` (for consistency)